src/ofono/provider_ofono.cpp

```diff
@@ -957,6 +977,50 @@ void Bridge::setup_sim(QString const &path)
         setup_stk(modem_path_);
 }
 
+void Bridge::setup_callManager(QString const &path)
+{
+    qDebug() << "Get call properties";
+    auto update = [this](QString const &n, QVariant const &v) {
```

Comment:
```update``` does not do anything, so maybe ```callManager_->GetProperties()```, ```VoiceCallManager::PropertyChanged()``` connection should be removed.
Or some property should be exposed?